### ISSUE TITLE
Incorrect input on LSTM diagram

### ISSUE THREAD

**Issue by mzarnecki** [ageron/handson-ml3#246]
### Enter the chapter number

Chapter 15. Flow processing using recurrent and convolutional networks

### Enter the page number

Page 541, Figure 15.12 LSTM cell (book edition in Polish)

### Enter the environment you are using to run the notebook

None

### Describe your issue

Hello @ageron,

I'm very glad I had the opportunity to read your book. It's rich in valuable content and references to many important publications.

1. I want to share some typo that I noticed. There is a possible inconsistency between the text and the diagram in Figure 15.12, which presents the LSTM cell.
In the text, it is explained that the current input vector x(t) and the previous short-term state h(t−1) are passed to the four fully connected layers. This is also consistent with the standard LSTM architecture.
However, in the diagram, the input on the left side of the LSTM cell seems to be labeled as x(t−1). Based on the explanation in the text, I believe this label should probably be h(t−1), while x(t) should be the current input entering from the bottom.

<img width="1152" height="1536" alt="Image" src="https://github.com/user-attachments/assets/8a81fc12-55e8-4d8a-828c-d0485765fb49" />

I also attach 2 more small inconsistencies.

2. I noticed first link on README.md for this repo is resolving in 404 "[Hands-on Machine Learning with Scikit-Learn, Keras and TensorFlow (3rd edition)](https://homl.info/er3):" - probably needs to be updated to https://www.oreilly.com/library/view/hands-on-machine-learning/9781098125967/

3. Chapter 18. Reinforcement Learning, page 648 (Polish edition) - footnote 10 about OpenAI describes it as non-profit research organization - it's true considering origins, but probably worth to update this description according to OpenAI evolution and current activity.
